app/models/graph/types/film.rb

```diff
@@ -0,0 +1,33 @@
+module Graph
+  module Types
+    Film = GraphQL::ObjectType.define do
+      name "Film"
+      description "A single film."
+
+      field :id, types.ID, "The ID of the Film."
+
+      field :title, types.String, "The title of this film"
+      field :episodeID, types.Int, "The episode number of this film.", property: :episode_id
+      field :openingCrawl, types.String,
+        "The opening paragraphs at the beginning of this film.", property: :opening_crawl
+
+      field :director, types.String, "The name of the director of this film."
+      field :producers, types[types.String] do
+        description "The name(s) of the producer(s) of this film."
+        resolve ->(film, _, _) { film.producer.delete(" \t\r\n").split(",") }
```

Comment:
Just curious, why did you need the `delete(" \t\r\n")`? Could we not have just done `.split(", ")`?